examples/python/get_open_access_pdf/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3.11-slim-bullseye
```

Comment:
Does this all work with 3.10? I kind of chose 3.10 as _new enough_ (called out in examples/ [README](https://github.com/allenai/s2-folks/blob/main/examples/python/README.md)) but don't want to go too new since folks in many environments may not be able to use the newest python versions without taking that on.

Reply:
Yes, it works just fine, I'll update the `Dockerfile`